css/main.css

```diff
@@ -0,0 +1,59 @@
+body{
+    text-align: center;
+    
+}
+.logo{
+    color: yellow;
+    background: black;} 
+} 
+h1{
+    
+    color: rgb(251, 255, 5);
+   
+}
+.list{
+    background: black;}
+li{
+    color:white;
+    list-style: none;
+    display: inline;
+    text-align: left;
+    text-transform: uppercase;
+}
+section{
```

Comment:
Try using flexbox here to move the inner box to the right

Reply:
I will try :)

thank you Jackie for every thing you are a Great Teacher.